### DISCUSSION TITLE
Symfony 5, Ajax response give Sfdump switching APP_ENV from dev to prod

### DISCUSSION THREAD

**Discussion by KravTeam** [symfony/symfony#44993] [Q&A] [answered]
When I switch from  APP_ENV=dev to prod, I get an empty response with undefine data from my controller JsonResponse.

I am on a track to fix this but need help:

When using postman, the request to my controller give me 2 things:

1:
<script> Sfdump = window.Sfdump || (function (doc) ...... </script>
=> it' what I can read when doing console.log(response);
2:
{"dataClubInfo":{"nomClub":"......}} with my preciouse data !
=> it's the response I get from my controller return new JsonResponse(["dataClubInfo"=>$retourClubTab]);

To resume on dev I have an objet with my data ( {dataClubInfo: {…}} ) and in prod I get no data but :<script> Sfdump = window.Sfdump )

Would someone be able to help me ?

**Comment by norkunas** [comment, marked as answer]
that means you left the `dump(..)` statement before the returning response or somewhere else, try to remove it

**Reply by KravTeam** [reply]
Thank you so much !
I've commented all the dump() before it's working now !
